mllib/src/main/scala/org/apache/spark/ml/classification/FMClassifier.scala

```diff
@@ -85,7 +85,6 @@ class FMClassifier @Since("3.0.0") (
    */
   @Since("3.0.0")
   def setFactorSize(value: Int): this.type = set(factorSize, value)
-  setDefault(factorSize -> 8)
```

Comment:
Where do the default params of `FMClassifier` move?

Reply:
I move setDefault for these params to FactorizationMachinesParams in FMRegressor when I fixed solver last time